tests/test_utils/test_validation.py

```diff
@@ -60,3 +62,38 @@ def test_validate_int_range(num: Any, low: int, high: int, valid: bool) -> None:
 )
 def test_normalize_input_list(input, result):
     assert normalize_input_list(input) == result
+
+
+valid_pubkey = "911e7c7fc980bcf5400980917ee92797d52d226768e1b26985fabaf5f214464059ab2d52170b0605f4c8e7a872cde436"
+valid_signature = (
+    "0x854053a7faebf4547ca3904ff14d896a994d5fb7289478681842fb72622364cd0cb4922170a370ea53234a734b47cd6"
+    "80c7edca86e8d796abd8eaeb8dd85d99e57c962c84d6642dff4b6e9bfb6d6df5fa22910c583f13135f5b2b43e4f95e8cf"
+)
+invalid_pubkey = "b54186e3dbdde180cc39f52e0cf4207c5745a50e2e8bd12f49b925f87682cab88ef108f60cf3ea1ac82b7c6fe796f5d6"
+invalid_signature = (
+    "0x8cceb99d17361031e01dfb6aa997554a35f60bcc8a106ac76fdea6f5a4780fb8b65b4cd827bca0c88b340508b69f577"
+    "50122db94c319aa05a0165e71b41f30c0982c415727b7e2387cce78d995acd54f038b743dc1426b0fb0d4783617d4fe6e"
+)
```

Comment:
I think it would be better to generate all these values (valid_pubkey, valid_signature, invalid_pubkey, invalid_signature) on the fly, during each test, with our current code from original mnemonics, chain settings, etc instead of using pre-baked hex data for which we don't know much about. Not only will it make the test more readable and easier to understand, it will likely increase the overall test coverage (testing more of the pubkey and signature generation part or retesting it again).

Reply:
This might be a difference of opinion with testing.
My idea was we are testing only `validate_signed_exit` and we should be explicit with our inputs and then have expected results similar to some tests in `test_ssz.py` and `test_crypto.py`.
If we expand this test scenario to utilize mnemonic, key, and exit generation, then I would consider that an integration test and should belong elsewhere. 

If we generated the logic on the fly, we would end up with something like for the setup
```
    mainnet_chain_settings = get_chain_setting("mainnet")
    holesky_chain_settings = get_chain_setting("holesky")
    mnemonic = get_mnemonic(language="en", words_path=WORD_LISTS_PATH)
    mnemonic2 = get_mnemonic(language="en", words_path=WORD_LISTS_PATH)
    correct_keystore= Credential(
        mnemonic=mnemonic,
        mnemonic_password="",
        index=0,
        amount=0,
        chain_setting=mainnet_chain_settings,
        hex_eth1_withdrawal_address=None,
    )
    wrong_chain_keystore = Credential(
        mnemonic=mnemonic,
        mnemonic_password="",
        index=0,
        amount=0,
        chain_setting=holesky_chain_settings,
        hex_eth1_withdrawal_address=None,
    )
    wrong_mnemonic_keystore = Credential(
        mnemonic=mnemonic2,
        mnemonic_password="",
        index=0,
        amount=0,
        chain_setting=mainnet_chain_settings,
        hex_eth1_withdrawal_address=None,
    )
    correct_exit = correct_keystore.generate_exit_transaction(0, 0)
    wrong_index_exit = correct_keystore.generate_exit_transaction(1, 0)
    wrong_epoch_exit = correct_keystore.generate_exit_transaction(0, 1)
    wrong_chain_exit = wrong_chain_keystore.generate_exit_transaction(0, 0)
    wrong_mnemonic_exit = wrong_mnemonic_keystore.generate_exit_transaction(0, 0)
```

Which is quite a bit.

How about we keep this as is and I create a separate integration test suite that does this dynamic generation so we can sweep through more of the code base?
Or if you feel unit tests are not an effective use of time for CI, I can replace the unit testing logic with something similar to the above.

Reply:
That is a good point. Let's leave it as is for now and see how to do other tests elsewhere.